.github/workflows/pr-python.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        python: [3.8]
+        python: [3.9]
```

Comment:
In your testing did you find that `python 3.9` built Lambda Layers work on a `python 3.8` Lambda function? Can you describe that in the PR description please?

Reply:
agreed :-)